### PR TITLE
test: logging level

### PR DESCRIPTION
**Pull request by Yongxuanzhang**
Test the logging level with os variable 'JINA_LOG_LEVEL' setting.
Related to https://github.com/jina-ai/jina/pull/1744.

### CODE REVIEW

File: tests/unit/logging/test_logging.py [hunk 1]
```diff
@@ -37,6 +38,23 @@ def test_logging_default():
             assert len(logger.handlers) == 2
 
 
+def test_logging_level():
+    with JinaLogger('test_logger', log_config=os.path.join(cur_dir, 'yaml/file.yml')) as logger:
+        log(logger)
+        assert logger.logger.level == LogVerbosity.from_string('INFO')
+
+    os.environ['JINA_LOG_LEVEL'] = 'SUCCESS'
```
Comment (JoanFM) [inline tests/unit/logging/test_logging.py:46]: add a try catch mechanism that ensures thr os env is properly deleted but at same time captures the assertion error

**Review by JoanFM — CHANGES_REQUESTED** [review]

File: tests/unit/logging/test_logging.py (hunk 1 above)
Comment (JoanFM) [reply tests/unit/logging/test_logging.py:46]: or split the test so that u can use fixtures to guarantee this

**Review by hanxiao — CHANGES_REQUESTED** [review]
this is not a valid test, as in your test assertion you write the same code as in the `logger`

```python
        if 'JINA_LOG_LEVEL' in os.environ:
            verbose_level = LogVerbosity.from_string(os.environ['JINA_LOG_LEVEL'])
        self.logger.setLevel(verbose_level.value)
```

This is the same as having the code `a=1` and then test it via `assert a==1`, it is almost meaningless in most cases.

The correct way is to test the emission of the logger, and see if it is suppressed given the current loglevel.

File: tests/unit/logging/test_logging.py (hunk 1 above)
Comment (Yongxuanzhang) [reply tests/unit/logging/test_logging.py]: Split the test and use fixtures in the latest commit. Thanks!

**Comment by Yongxuanzhang** [comment]
> this is not a valid test, as in your test assertion you write the same code as in the `logger`
> 
> ```python
>         if 'JINA_LOG_LEVEL' in os.environ:
>             verbose_level = LogVerbosity.from_string(os.environ['JINA_LOG_LEVEL'])
>         self.logger.setLevel(verbose_level.value)
> ```
> 
> This is the same as having the code `a=1` and then test it via `assert a==1`, it is almost meaningless in most cases.
> 
> The correct way is to test the emission of the logger, and see if it is suppressed given the current loglevel.

Revised in the latest commit.

**Pull request merged**